infra/Pos/Network/CLI.hs

```diff
@@ -329,7 +328,12 @@ fromPovOf cfg@NetworkConfigOpts{..} allPeers =
             hasKademlia   = M.filter nmKademlia (Y.allStaticallyKnownPeers allPeers)
             selfKademlia  = M.member self hasKademlia
             otherKademlia = M.delete self hasKademlia
-            allKademlia   = M.keys otherKademlia ++ if selfKademlia then [self] else []
+            -- Linter claims that
+            --   [self | selfKademlia]
+            -- is more readable than
+            --   if selfKademlia then [self] else []
+            allKademlia   = M.keys otherKademlia ++ [self | selfKademlia]
```

Comment:
Heh, that's just Neil's opinion. It's a little idiom that is nice if you know it, but as with all idioms you have to know it. So you want to turn off this linter warning with a pragma, I would not complain. (Though personally I think it's quite a neat idiom)